uniflow/op/extract/split/markdown_header_splitter.py

```diff
@@ -33,8 +33,12 @@ class MarkdownHeaderSplitter(Op):
         ("##", "Header 2"),
         ("###", "Header 3"),
         ("####", "Header 4"),
+        ("#####", "Header 5"),
+        ("######", "Header 6"),
     ]
 
+    lines_with_metadata: List[LineType] = []
```

Comment:
one potential issue for this class variable is that if `MarkdownHeaderSplitter` is used twice in two different instance, it can be problematic, I would suggest include it in a __init__ method with super().__init__() as well.